### PR TITLE
docs (dev): expanded tailwind component docs

### PR DESCRIPTION
**Pull request by davidruvolo51**
What are the main changes you did:
- Added information on the component library
- Expanded docs on building frontend apps
- Added vscode extension recommendations for markdown and linting configuration

how to test:
 - View developer guidelines: https://preview-emx2-pr-4079.dev.molgenis.org/apps/docs/#/molgenis/dev_guidelines
 - View app development guide: https://preview-emx2-pr-4079.dev.molgenis.org/apps/docs/#/molgenis/dev_apps

todo:
- [x] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

is a part of https://github.com/molgenis/GCC/issues/572 but contributors.md will be done elsewhere

### CODE REVIEW

**Comment by davidruvolo51** [comment]
In the developing frontend apps guide, I removed the section on creating a component library. I don't think it's necessary to have it in this guide. We would also like to avoid having too many component libraries.

File: apps/tailwind-components/README.md [hunk 1]
```diff
@@ -1,56 +1,101 @@
...
+The `tailwind-components` library is a reimplementation of the existing emx2 components using nuxt and tailwind. This allows us to style build interfaces using a standard set of components to ensure all interfaces have a consistent layout and design.
 
-## Setup
+## About the components
 
-Make sure to install the dependencies:
+All components are stored in the `components` folder and stories are located in `pages`. To view the components, take a look at the [components showcase](https://emx2.dev.molgenis.org/apps/tailwind-components) or run the `yarn dev` command.
 
-```bash
+In the component folder, we've arranged the components into categories. To use the components, you will need to prefix the component name using the name of the subfolder. For example, if you wanted to use the checkbox component (`components/input/Checkbox.vue`), you would write the component as `<InputCheckbox>`. There are some folders that do not to be prefixed with the subfolder. This is described in the table below.
```
Comment (chinook25) [inline apps/tailwind-components/README.md:9]: are not prefixed?

File: .gitignore [hunk 2]
```diff
@@ -29,7 +29,7 @@ derby.log
 .idea/*
 .idea/fileTemplates/*/
 !.idea/fileTemplates/
-.vscode/*
+# .vscode/*
```
Comment (chinook25) [inline .gitignore:32]: might as well remove it

File: apps/tailwind-components/README.md [hunk 3]
```diff
@@ -1,56 +1,101 @@
...
 ```
 
-## Production
+### Setup
 
-Build the application for production:
+We use [yarn workspaces](https://yarnpkg.com/features/workspaces) to manage dependencies across all applications. On the first time, install the dependencies.
 
 ```bash
+yarn install
```
Comment (chinook25) [inline apps/tailwind-components/README.md:63]: just `yarn` is enough. Not sure, but I feel adding 'install' is a npm leftover?

**Review by chinook25 — APPROVED** [review]

**Pull request merged**